sdk/metric/metricdata/metricdatatest/assertion.go

```diff
@@ -53,6 +52,19 @@ type Datatypes interface {
 	// Aggregation and Value type from metricdata are not included here.
 }
 
+// TestingT is an interface that implements `testing.T`, but without the
+// private method of `testing.TB`, so other testing packages can rely on it as
+// well.
+type TestingT interface {
```

Comment:
Optional:
We could add a comment that this should only change by adding methods that exist on `*Testing.T`.